stock_lot_production_date/models/stock_lot.py

```diff
@@ -8,5 +8,6 @@ class StockProductionLot(models.Model):
     _inherit = "stock.lot"
 
     production_date = fields.Datetime(
-        help="This is the date when the goods with this lot/serial number have been produced."
+        help="This is the date when the goods with this \
```

Comment:
Revert this change to original way, please

The original line is good for our guidelines

Reply:
@moylop260 We have issue with pre-commit in this line so we used the "\", now we have removed that. Please check once again.

Reply:
What issue?

I mean, the length of the line is good for the guidelines defined but not sure what was the problem?

Reply:
@moylop260 The error we got is 
ruff.....................................................................Failed
hook id: ruff
exit code: 1

stock_lot_production_date/models/stock_lot.py:11:89: E501 Line too long (94 > 88)
Found 1 error.

Actual length of the line in our file is 94 but in the .ruff.toml file it is mentioned that  "E501",  # line too long (default 88).

Reply:
@sbidoul we had changed to length 88 or it was a default hidden?

Reply:
I don't think we changed anything in that area recently. Probably ruff and black+flake8 have a slightly different interpretation.